packages/eas-cli/src/build/android/prepareJob.ts

```diff
@@ -56,7 +56,7 @@ async function prepareJobCommonAsync(
             dataBase64: credentials.keystore.keystore,
             keystorePassword: credentials.keystore.keystorePassword,
             keyAlias: credentials.keystore.keyAlias,
-            keyPassword: credentials.keystore.keyPassword,
+            keyPassword: credentials.keystore.keyPassword ?? '',
```

Comment:
it's fine as a temporary solution, but the proper approach would be to first add support in EAS Build and merge it after